assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -439,3 +452,23 @@ bool EntityTreeSendThread::traverseTreeAndBuildNextPacketPayload(EncodeBitstream
 #endif // SEND_SORTED_ENTITIES
 }
 
+void EntityTreeSendThread::editingEntityPointer(const EntityItemPointer entity) {
+    if (entity) {
+        if (_entitiesInQueue.find(entity.get()) == _entitiesInQueue.end() && _knownState.find(entity.get()) != _knownState.end()) {
+            bool success = false;
+            AACube cube = entity->getQueryAACube(success);
+            if (success) {
+                float priority = _conicalView.computePriority(cube);
+                _sendQueue.push(PrioritizedEntity(entity, priority, true));
+                _entitiesInQueue.insert(entity.get());
+            } else {
+                _sendQueue.push(PrioritizedEntity(entity, PrioritizedEntity::WHEN_IN_DOUBT_PRIORITY, true));
+                _entitiesInQueue.insert(entity.get());
+            }
+        }
+    }
+}
```

Comment:
Hrm... whenever anything changes: for each sendThread we check to see if the changed thing is not yet on the _`sendQueue` but is listed in the _`knownState`, and if so: add to the _`sendQueue`.  This seems too expensive for a couple reasons:

(1) if the thing is in view then the normal traversal scan will find it
(2) there is work done here that could maybe be done only once per change rather than: once per change, per sendThread.